src/ui/public/indices/get_indices.js

```diff
@@ -0,0 +1,27 @@
+import { reduce, size } from 'lodash';
+
+const getIndicesFromResponse = json => {
+  return reduce(json, (list, { aliases }, indexName) => {
+    list.push(indexName);
+    if (size(aliases) > 0) {
+      list.push(...Object.keys(aliases));
+    }
+    return list;
+  }, []);
+};
+
+export function IndicesGetIndicesProvider(esAdmin) {
+  return async function getIndices(query) {
+    try {
+      const aliasesJson = await esAdmin.indices.getAlias({ index: query });
```

Comment:
Sorry, I means that the `esAdmin.indices.getAlias()` function takes an `allowNoIndices` param: https://www.elastic.co/guide/en/elasticsearch/client/javascript-api/current/api-reference.html#api-indices-getalias